lib/ts/recipe/oauth2/types.ts

```diff
@@ -48,6 +48,22 @@ export type APIOptions = {
 };
 
 export type RecipeInterface = {
+    getOAuth2Clients(
+        input: GetOAuth2ClientsInput,
+        userContext: UserContext
+    ): Promise<
+        | {
+              status: "OK";
+              clients: Array<OAuth2Client>;
+              nextPageToken?: string;
```

Comment:
Please rename this to `paginationToken` token both in the input and the return type to match `getUsersNewestFirst`

Reply:
done